src/GNDStk.hpp

```diff
@@ -17,6 +17,7 @@
 
 // miscellaneous
 #include <algorithm>
+#undef NDEBUG
```

Comment:
No. Don't do this.

If you undefine it, it will propagate everywhere and could/will have unintended consequences. The Eigen library for instance is a header only library that uses a lot asserts to check calculations in Debug mode. As a result, it is a lot faster in Release mode where -DNDEBUG is used as a compiler flag.

Reply:
OK, I changed it.